Framework/Core/include/Framework/AnalysisTask.h

```diff
@@ -71,12 +71,20 @@ struct WritingCursor<soa::Table<PC...>> {
   void operator()(T... args)
   {
     static_assert(sizeof...(PC) == sizeof...(T), "Argument number mismatch");
+    ++mCount;
     cursor(0, extract(args)...);
   }
 
+  /// Last index inserted in the table
+  size_t lastIndex()
+  {
+    return mCount;
```

Comment:
Looking at the index returned when going through collections,  just wondering whether this should not be:
```suggestion
    return mCount - 1;
```

Reply:
yes, it should.

Reply:
i will initialize to -1